RoomsExpanded/Translations/vi.po

```diff
@@ -0,0 +1,520 @@
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.LABORATORY.NAME
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.LABORATORY.NAME"
+msgid "Laboratory"
+msgstr "Phòng Nghiên Cứu"
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.LABORATORY.TOOLTIP
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.LABORATORY.TOOLTIP"
+msgid "Conducting research in a designated room proves to be more efficient."
+msgstr "Thực hiện nghiên cứu một cách hiệu quả hơn và nhanh hơn."
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.MUSEUM.EFFECT
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.MUSEUM.EFFECT"
+msgid " - Morale bonus"
+msgstr " - Phần thưởng tinh thần"
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.MUSEUM.NAME
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.MUSEUM.NAME"
+msgid "Museum"
+msgstr "Viện Bảo Tàng"
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.MUSEUM.TOOLTIP
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.MUSEUM.TOOLTIP"
+msgid "It used to be a storehouse, before Meep confused Pedestal content with art."
+msgstr "Nó từng là một nhà kho, trước khi Meep nhầm lẫn nội dung của Pedestal với nghệ thuật."
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.NURSERY.EFFECT
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.NURSERY.EFFECT"
+msgid " - Chance for additional seeds"
+msgstr " - Cơ hội cho thêm hạt giống"
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.NURSERY.NAME
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.NURSERY.NAME"
+msgid "Plant Nursery"
+msgstr "Vườn Ươm Cây"
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.NURSERY.TOOLTIP
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.NURSERY.TOOLTIP"
+msgid "Enriched environment improves plant development."
+msgstr "Môi trường phong phú cải thiện sự phát triển của thực vật."
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.PRIVATEROOM.EFFECT
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.PRIVATEROOM.EFFECT"
+msgid " - Morale bonus"
+msgstr " - Phần thưởng tinh thần"
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.PRIVATEROOM.NAME
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.PRIVATEROOM.NAME"
+msgid "Private Bedroom"
+msgstr "Phòng Ngủ Riêng Tư"
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.PRIVATEROOM.TOOLTIP
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.PRIVATEROOM.TOOLTIP"
+msgid "For duplicants who want to have some privacy."
+msgstr "Đối với những Đệ muốn có một số quyền riêng tư."
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.SHOWERROOM.EFFECT
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.SHOWERROOM.EFFECT"
+msgid " - Shorter shower time"
+msgstr " - Thời gian tắm ngắn hơn"
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.SHOWERROOM.NAME
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.SHOWERROOM.NAME"
+msgid "Shower Room"
+msgstr "Phòng tắm"
+
+#. RoomsExpanded.STRINGS.ROOMS.TYPES.SHOWERROOM.TOOLTIP
+msgctxt "RoomsExpanded.STRINGS.ROOMS.TYPES.SHOWERROOM.TOOLTIP"
+msgid "Just entering such a nice bathroom make duplicants feel cleaner."
+msgstr "Đệ chỉ cần bước vào phòng tắm đẹp sẽ cảm thấy sạch sẽ hơn."
+
+#. RoomsExpanded.STRINGS.TRANSLATION.AUTHOR.NAME
+msgctxt "RoomsExpanded.STRINGS.TRANSLATION.AUTHOR.NAME"
+msgid "pether.pg"
+msgstr ""
```

Comment:
Feel free to sign your name here, I print it as a thank you message :)

Reply:
Oh, thank